### DISCUSSION TITLE
Things I had to go do to get Plover to work with Polygot keyboard on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by fearofcode** [opensteno/plover#1609] [General]
Sharing this for myself and anyone else in this situation:

- Install libfuse2: `sudo apt install libfuse2`
- Install qt. If you try to the old package `qt5-default` that every search result says to use, it won't work, the package doesn't exist anymore. Instead run `sudo apt install -y qtcreator qtbase5-dev qt5-qmake cmake`.
- Now the weirdest one, that I think is specific to this keyboard: I was getting a `PermissionDenied` error when trying to add the Polyglot keyboard. The fix I did was to add myself to the `dialout` group so that it could talk to serial devices??? ``sudo usermod -a -G dialout `whoami``, log out, log in. Yes, I tried running the app with `sudo`. The app threw a stack trace and would not boot.

I did not expect this since AppImages are described as being self-contained.

**Comment by user202729** [comment]
Actually the dialout part is in https://github.com/openstenoproject/plover/wiki/Installation-Guide#allowing-access-to-serial-ports .
